core/Cargo.toml

```diff
@@ -15,6 +15,7 @@ smallvec = "1.0"
 petgraph = { version = "0.5", optional = true }
 thread-id = { version = "3.2.0", optional = true }
 backtrace = { version = "0.3.2", optional = true }
+instant = "0.1"
```

Comment:
I like the concept of `instant` but I am concerned that it is pulling in a lot of dependencies (through `time`) on non-WASM platforms. Looking at the code of that crate, it seems that `time` is only used to provide a separate `now` function. Could that be made optional so `parking_lot` doesn't have to depend on it?

Reply:
hm, `time` is optional in `instant` by default (it needs the `"now"` feature enabled).
Is that sufficient? Also, should I wire through the `"now"` feature in `parking_lot`?

Reply:
Ah sorry, I missed that. There's no need to wire through `"now"` since parking_lot doesn't use it.